jest.config.ts

```diff
@@ -1,3 +1,15 @@
-export default {
-  preset: 'jest-expo/universal',
+import type { Config } from 'jest';
+
+const config: Config = {
+  projects: [
+    { preset: 'jest-expo/android' },
+    { preset: 'jest-expo/ios' },
+    {
+      preset: 'jest-expo/web',
+      testMatch: ['**/__tests__/**/*.test.(web|node).ts?(x)'],
+    },
```

Comment:
`web` environment is only used for test files with `.node.ts` or `.web.ts` extensions. Only specific components like the feed parser should run here (environment-agnostic). All other tests run on android and ios environments.